lib/vector-core/src/transform/mod.rs

```diff
@@ -271,13 +280,14 @@ impl TransformOutputs {
             let count = buf.len();
             let byte_size = buf.size_of();
             buf.send(self.named_outputs.get_mut(key).expect("unknown output"))
-                .await;
+                .await?;
             emit(EventsSent {
                 count,
                 byte_size,
                 output: Some(key.as_ref()),
             });
         }
+        Ok(())
```

Comment:
```suggestion

        Ok(())
```

Seemingly been formatting it this way?